app/components/QuestionRow/QuestionRow.jsx

```diff
@@ -41,14 +41,34 @@ const QuestionRow = (props) => {
   const { locations } = useLoaderData();
   const isUpdated = false;
   const pinForm = useRef();
+  const enabledForm = useRef();
   const submit = useSubmit();
 
   const [searchParams] = useSearchParams();
 
+  const handleStatusClick = () => {
+    let url = '/?index';
+    if (!isFromList) {
+      url = `/questions/${question.question_id}`;
+      const urlSearchParam = searchParams.get('order');
+      url = urlSearchParam !== null ? `${url}?order=${urlSearchParam}` : url;
+    } else {
+      searchParams.forEach((value, key) => {
+        url += value ? `&${key}=${value}` : '';
+      });
+    }
+    const data = new FormData(enabledForm.current);
+    data.set('action', ACTIONS.ENABLED_ACTION);
+    data.set('questionId', question.question_id);
+    data.set('enabledValue', !question.is_enabled);
+
+    submit(data, { method: 'post', action: url });
```

Comment:
Can you please add `replace: true` to avoid the following behavior. https://github.com/wizeline/wize-q-remix/issues/118